lib/safe_json_serializer.rb

```diff
@@ -0,0 +1,20 @@
+module SafeJSONSerializer
```

Comment:
why `module` instead of `class`? using `class` will remove additional `module_function`

Reply:
I was imitating what is implemented for the other serializers.  I can try a class.